.rubocop.yml

```diff
@@ -23,7 +23,7 @@ Metrics/MethodLength:
     - 'enterprise/lib/captain/agent.rb'
 
 RSpec/ExampleLength:
-  Max: 25
+  Max: 50
```

Comment:
Let's revert this. For the example in the PR specifically we can either extract the constants or params etc outside the example, or just add `rubocop:disable` for that rule on a case-to-case basis